sqflite/android/src/main/java/com/tekartik/sqflite/Database.java

```diff
@@ -391,25 +390,14 @@ private boolean executeOrError(Operation operation) {
         if (LogLevel.hasSqlLevel(logLevel)) {
             Log.d(TAG, getThreadLogPrefix() + command);
         }
-        boolean operationInTransaction = Boolean.TRUE.equals(operation.getInTransactionChange());
-
+        Boolean operationInTransaction = operation.getInTransactionChange();
         try {
             getWritableDatabase().execSQL(command.getSql(), command.getSqlArguments());
-
-            // Success handle inTransaction change
-            if (operationInTransaction) {
```

Comment:
(note to myself) I will likely rename operationInTransaction to operationInTransactionChange